.copywrite.hcl

```diff
@@ -0,0 +1,17 @@
+schema_version = 1
+
+project {
+  license        = "MPL-2.0"
+  copyright_year = 2021
+
+  # (OPTIONAL) A list of globs that should not have copyright/license headers.
+  # Supports doublestar glob patterns for more flexibility in defining which
+  # files or folders should be ignored
+  header_ignore = [
+    "**/*.yml",
+    "**/*.cjs",
+    "**/*rc.js",
+    "**/dist/**",
+    "packages/flight-icons/svg-react/*",
+  ]
```

Comment:
This seems like a good list! Are the `.cjs` and `.yml` files representative of anything substantial, or just GHA configs, etc?

One quick note - if you are running `copywrite` locally, it may be advantageous to add `node_modules/**` to the config for performance. The tooling is pretty quick (85k files in about 35s), but `node_modules` is a blackhole. That said, I'm considering having the tool automatically skip it, so leaving it out is fine (plus that isn't checked into git so the automated scans won't care).

Reply:
For `.csj` we have only one file, and don't foresee others `.yarn/releases/yarn-3.2.4.cjs`
For `.yml` there are two: `.github/dependabot.yml` and `.yarnrc.yml`
Good call for the `node_modules`, can add it now (as you said, for performance) and remove it in future if skipped by the tool.